tools/autotools/tsk_gettimes.cpp

```diff
@@ -88,9 +88,10 @@ TSK_FILTER_ENUM
 TskGetTimes::filterFs(TSK_FS_INFO * fs_info)
 {
     TSK_TCHAR volName[32];
-    if (m_curVolAddr > -1)
+    if (m_curVolAddr > -1) {
         TSNPRINTF(volName, 32, _TSK_T("vol%d/"),m_curVolAddr);
-    else 
+    }
+    else {
         volName[0] = '\0';
 
```

Comment:
Looks like the indentation on this file is wrong.  The closing brace should go here.